### DISCUSSION TITLE
Simple check-test using renderOnly

### DISCUSSION THREAD

**Discussion by grzegorzCieslik95** [storybookjs/storybook#26618] [Help]
### Summary

Hi, I want to write a simple test for my CI where i want to check if all of my stories render correctly (without any errors). I tried to use renderOnly but in documentation, I cant find any examples for this. 

Thanks for help

**Comment by valentinpalkovic** [comment]
Hi @grzegorzCieslik95 

Thank you for reporting this issue!

I think https://storybook.js.org/addons/@storybook/test-runner is the right thing to set up. If your components don't have play functions, it just checks whether your components are rendering without any errors. Please try it out and let me know if it helps!
